extensions/wikia/Email/EmailImageHelper.class.php

```diff
@@ -13,17 +13,38 @@ class ImageHelper {
 	const ICON_CACHE_TTL = 86400;
 
 	static public $icons = [
-		'Wikia',
-		'Comics',
-		'Games',
-		'Movies',
-		'Lifestyle',
-		'Music',
-		'Books',
-		'TV',
-		'Twitter',
-		'Facebook',
-		'YouTube',
+		'FandomLogoHeader' => [
+			'name' => 'Hero-Logo-2x',
+			'extension' => 'png'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "extension" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)